### PR TITLE
MuhammadFareezIqmal.IIUMSchedule version 0.1.0.7

### PR DESCRIPTION
**Pull request by iqfareez**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/68784)

### CODE REVIEW

**Comment by iqfareez** [comment]
Hi. How can I solve the issue:
> An anti-virus product reports an infection in the installer

**Comment by iqfareez** [comment]
image

No issue on virustotal.com

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.7/MuhammadFareezIqmal.IIUMSchedule.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 0.1.0.7
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/iqfareez/iium_schedule/releases/download/1.0.0-pre.7%2B10/iiumschedulesetup.exe
+  InstallerSha256: 4DD175BC6C0235503ADEB0743FB59CA4E53BBDDD4EED9AB83A849163165F849D
```
Comment (ItzLevvie) [inline manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.7/MuhammadFareezIqmal.IIUMSchedule.installer.yaml:10]:
```suggestion
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/iqfareez/iium_schedule/releases/download/1.0.0-pre.7%2B10/iiumschedulesetup.exe
  InstallerSha256: 4DD175BC6C0235503ADEB0743FB59CA4E53BBDDD4EED9AB83A849163165F849D
  InstallerSwitches:
    Custom: /CURRENTUSER
  Scope: user
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/iqfareez/iium_schedule/releases/download/1.0.0-pre.7%2B10/iiumschedulesetup.exe
  InstallerSha256: 4DD175BC6C0235503ADEB0743FB59CA4E53BBDDD4EED9AB83A849163165F849D
  InstallerSwitches:
    Custom: /ALLUSERS
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.7/MuhammadFareezIqmal.IIUMSchedule.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 0.1.0.7
+PackageLocale: en-US
+Publisher: Muhammad Fareez Iqmal
+PackageName: IIUM Schedule
```
Comment (ItzLevvie) [inline manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.7/MuhammadFareezIqmal.IIUMSchedule.locale.en-US.yaml:8]:
```suggestion
PackageName: IIUM Schedule (Preview)
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**